.eslintrc

```diff
@@ -1,3 +1,13 @@
 {
-    "extends": "ebay"
-}
\ No newline at end of file
+    "extends": "ebay",
+    "rules": {
+         "import/order": [2, {"groups": ["builtin", "external", "internal", "parent", "sibling", "index"]}],
+        "no-param-reassign": 0
+    },
+    "parserOptions": {
+        "ecmaVersion": 2017,
```

Comment:
Note to self: Changes in this PR will require a major version change to `^1.0.0`